Gems/Atom/Tools/MaterialCanvas/Code/Source/Window/MaterialCanvasMainWindow.cpp

```diff
@@ -220,6 +220,12 @@ namespace MaterialCanvas
         Base::PopulateSettingsInspector(inspector);
     }
 
+    void MaterialCanvasMainWindow::OnSettingsDialogClosed()
+    {
+        AtomToolsFramework::SetSettingsObject("/O3DE/Atom/GraphView/ViewSettings", m_graphViewSettingsPtr);
+        Base::OnSettingsDialogClosed();
```

Comment:
is this resetting the value to default on shutdown/close? If it is that could do with a comment.